tests/components/mqtt/test_config_flow.py

```diff
@@ -1390,10 +1392,6 @@ async def test_disable_birth_will(
         mqtt.CONF_WILL_MESSAGE: {},
     }
 
-    await hass.async_block_till_done()
-    # assert that the entry was reloaded with the new config
-    assert mock_reload_after_entry_update.call_count == 1
-
```

Comment:
Why is this removed. I understand we change the way we reload, but I want to make sure the entry is updated here.